polaris-pipeline/Common/Dto/Tracker/TrackerDto.cs

```diff
@@ -29,12 +29,6 @@ public class TrackerDto
         [JsonProperty("retrieved")]
         public float? Retrieved { get; set; }
 
-        [JsonProperty("pdfsGenerated")]
```

Comment:
`Indexed` and `PdfsGenerated` need to be removed from the gateway `TrackerDto` model also.  It doesn't look like the UI knows about `Indexed`, `PdfsGenerated` or even `Logs` so that is ok.

Reply:
The TrackerDto is in the common library, therefore shared by the Gateway and Pipeline. I did find a couple of entity files that referenced the removed log entities, these have been removed.